openpower/package/ppe42-gcc/Config.in

```diff
@@ -1,7 +1,7 @@
 config BR2_PACKAGE_HOST_PPE42_GCC
         bool "ppe42-gcc"
-        default y if (BR2_OPENPOWER_PLATFORM && BR2_OPENPOWER_POWER9)
 	select BR2_PACKAGE_HAS_PPE42_TOOLCHAIN
+	select BR2_PACKAGE_PPE42_BINUTILS
```

Comment:
Why?

Reply:
This may be related with the hack mentioned above, but I feel like keeping this one because ppe42-gcc.mk depends on host-ppe42-binutils, but it's config file doesn't explicitly select it (it was always selected due to the common criteria before)